### ISSUE TITLE
Accessibility Challenges: 

### ISSUE THREAD

**Issue by Nasratullah-Shafiq** [Nasratullah-Shafiq/Todo-app#19]
Failing to make the app usable for people with disabilities, such as lack of screen reader support.
